cli/commands.go

```diff
@@ -481,6 +490,7 @@ func AddNode(c *cli.Context) error {
 		AutoRestart:       false,
 		ClusterName:       clusterName,
 		Env:               nil,
+		Labels:            nil,
```

Comment:
Why shouldn't we add the labels to the server/master node?
Maybe we can integrate the label feature with our node-selectors? (like for volumes/ports)

Reply:
Good point. I wasn't happy about that worker-only feature, especially for single node deployments.

For using node selectors, the problem is that labels could contains `@`.

Do you prefer the selector solution over using something like `--worker-label` and `--server-label` ?

Reply:
True, that's a nice pitfall there :thinking: 
I'd definitely be happier with a node-selector based solution for the sake of consistency (other flags use them already) and flexibility (select one or more nodes by name or role).
If this is too complicated though, then I'm also OK with your proposed alternative solution of having to distinct flags.

Reply:
Ok, I can to re-use the same logic you used for ports.

I'll check if I can share some code between ports and labels without changing too many things.